docs/OpenBSD.md

```diff
@@ -48,23 +48,14 @@ Install OpenBSD over PXE
   	menu default
   	kernel OpenBSD/amd64/pxeboot.0
   ```
```

Comment:
@pkubaj consider adding some earlyprint over COM. I have no idea if the booting is proceeding or not when I run that menu option.

Reply:
Do you mean the option in PXELINUX or the autoboot option in iPXE? Generally, both load very quickly and it's usually you environment (dhcpd or tftpd configuration) which is bad.